INSTALL.md

```diff
@@ -2,24 +2,25 @@
 
 Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
```

Comment:
There is just one helm chart, so using plural is confusing and maybe refer to the charts directory and the README there as well.

Reply:
This has been taken care.